### PR TITLE
NOISSUE - Fix hardcoded env var values

### PR DESCRIPTION
**Pull request by darkodraskovic**

### CODE REVIEW

File: docker/ssl/Makefile [hunk 1]
```diff
@@ -11,29 +11,29 @@ all: clean_certs ca server_crt
 # CA name and key is "ca".
 ca:
 	openssl req -newkey rsa:2048 -x509 -nodes -sha512 -days 1095 \
-				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=localhost/O=Mainflux/OU=IoT/emailAddress=info@mainflux.com"
+				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=$(CN)/O=$(O)/OU=$(OU)/emailAddress=$(EA)"
```
Comment (mteodor) [inline docker/ssl/Makefile:14]: Docs that describe the use of this script should reflect this change
so we should put something like
`make ca CN=localhost O=Mainflux OU=IoT`

**Review by manuio — DISMISSED** [review]
LGTM

File: docker/ssl/Makefile [hunk 2]
```diff
@@ -11,29 +11,29 @@ all: clean_certs ca server_crt
...
-	# Remove CSR.
+
+# Remove CSR.
 	rm $(CRT_LOCATION)/mainflux-server.csr
 
 thing_cert:
-	# Create mainflux server key and CSR.
+# Create mainflux server key and CSR.
 	openssl req -new -sha256 -newkey rsa:4096 -nodes -keyout $(CRT_LOCATION)/$(CRT_FILE_NAME).key \
 				-out $(CRT_LOCATION)/$(CRT_FILE_NAME).csr -subj "/CN=$(KEY)/O=$(O)/OU=$(OU)/emailAddress=$(EA)"
```
Comment (blokovi) [inline docker/ssl/Makefile:31]: Can you also change default value for `KEY`, now it is `default`. I would use some value in format of Mainflux Thing Key.

File: docker/ssl/Makefile [hunk 3]
```diff
@@ -11,29 +11,29 @@ all: clean_certs ca server_crt
 # CA name and key is "ca".
 ca:
 	openssl req -newkey rsa:2048 -x509 -nodes -sha512 -days 1095 \
-				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=localhost/O=Mainflux/OU=IoT/emailAddress=info@mainflux.com"
+				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=$(CN)/O=$(O)/OU=$(OU)/emailAddress=$(EA)"
 
 # Server cert and key name is "mainflux-server".
 server_cert:
-	# Create mainflux server key and CSR.
+# Create mainflux server key and CSR.
```
Comment (drasko) [inline docker/ssl/Makefile]: Donot move these comments to the beginning of the line - they belong to indented code.

**Review by drasko — CHANGES_REQUESTED** [review]

File: docker/ssl/Makefile (hunk 2 above)
Comment (blokovi) [inline docker/ssl/Makefile]: Can you also change default value for KEY, now it is default. I would use some value in format of Mainflux Thing Key.

**Review by blokovi — CHANGES_REQUESTED** [review]

File: docker/ssl/Makefile (hunk 3 above)
Comment (darkodraskovic) [reply docker/ssl/Makefile]: Besides, it will be treated as a command by shell and it will be echoed as a comment.

Comment (drasko) [reply docker/ssl/Makefile]: `Blank lines and lines of just comments may appear among the recipe lines; they are ignored.` - they should be completely ignored.

Did you try and verify if the shell echoes these in the case they are indented and not in the case they are not?

**Review by dborovcanin — APPROVED** [review]

File: docker/ssl/Makefile (hunk 1 above)
Comment (darkodraskovic) [reply docker/ssl/Makefile:14]: Did a PR.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**